## Discussion

### Epistasis between germline mutator alleles

We discovered a gene on chromosome 6 that enhances a specific type of genetic mutation in the BXDs, a group of laboratory mice bred from the DBA/2J and C57BL/6J strains.
The DBA/2J alleles at this gene do not impact the mutation rates in mice with "normal" C57BL/6J alleles at another known mutation gene on chromosome 4.
However, mice with DBA/2J alleles at both genes show even higher mutation rates compared to those with DBA/2J alleles at only the chromosome 4 gene (Figure 1).
While interactions between mutation genes have been observed in yeast and human cell lines, this is the first instance, to our knowledge, in a whole-animal system.

We found interactions between mutator alleles in a controlled laboratory setting using model organisms [@PMID:33472028].
The breeding conditions likely reduced the impact of natural selection on harmful alleles [@doi:10.1146/annurev.ecolsys.39.110707.173437], potentially leading to the fixation of high-impact mutator alleles not commonly found in the wild.
Without pinpointing the origin of the chromosome 6 mutator allele, we cannot determine if it arose in a laboratory colony or a wild *Mus musculus* population.
If the chromosome 6 mutator allele has even a slight negative impact on fitness, it may have originated in captivity.
The relaxation of selection pressure could quickly increase mutation rates, as seen in a recent discovery in a rhesus macaque research colony [@doi:10.1101/2023.03.27.534460] and in domesticated animals with higher mutation rates compared to wild counterparts [@PMID:36859541].
While we have not definitively identified the causal variant for the chromosome 6 mutator locus, we propose that nonsynonymous or regulatory variants in the DNA glycosylase *Ogg1* are strong candidates.


### Protein-coding genes that may underlie the chromosome 6 mutator locus

Five genes involved in DNA repair overlap with the C>A mutator locus on chromosome 6.
These genes are Ogg1, Setmar, Fancd2, Rad18, and Mbd4.
Ogg1 is a glycosylase that removes the oxidative DNA lesion 8-oxoguanine.
Setmar is a histone methyltransferase involved in repairing double-stranded breaks.
Fancd2 and Rad18 are also involved in DNA repair processes.
Mbd4 is located just outside the confidence interval on chromosome 6.
While we cannot definitively identify which gene is responsible for the C>A mutator phenotype, Ogg1 is considered the most likely candidate.
Ogg1 is in the same repair pathway as Mutyh, which is associated with the chromosome 4 mutator locus.
Additionally, Ogg1 has genetic differences between the C57BL/6J and DBA/2J strains and is regulated by cis-eQTLs in various tissues within the BXD cohort.

The coding sequences of *Setmar* in C57BL/6J and DBA/2J mice have two missense variants, one of which is predicted to be harmful according to computational tools.
The primate version of *SETMAR* plays a role in repairing double-strand breaks in DNA through a combination of a SET methyltransferase domain and a *Mariner*-family transposase domain.
However, the murine version of *Setmar* lacks the transposase domain, suggesting it may not be responsible for the interaction between mutator loci on chromosomes 4 and 6 in BXD mice.
We also found no significant genetic variations in other DNA repair genes like *Fancd2* or *Rad18* that could explain the increased mutation rate.
In fact, an analysis of gene expression in different tissues showed that *D* alleles actually led to higher expression of *Fancd2* in gastrointestinal tissue.


<!-- We also note that *FANCD2* deficiency in human cancer can lead to increased rates of *de novo* structural variation [@PMID:36450981]. -->

### An *Ogg1* mutator allele might impair the excision of 8-oxoguanine lesions

*Ogg1* and *Mutyh* are both involved in the base-excision repair (BER) pathway, which responds to oxidative DNA damage and prevents C>A mutations.
*Ogg1* is responsible for recognizing and removing 8-oxoguanine lesions that are paired with cytosine nucleotides.
If these lesions are not removed before DNA replication, adenine may be incorrectly inserted opposite 8-oxoG.
In such cases, *Mutyh* can remove the mispaired adenine, leading to the formation of a C:8-oxoG base pair that is repaired by other BER proteins.
Defects in this repair process result in elevated C>A mutation rates.
For instance, mice lacking *Ogg1*, *Mutyh*, and *Mth1* accumulate excess 8-oxoG in their gonadal cells, with almost all germline mutations being C>A transversions.
Mutations and loss-of-heterozygosity in *Ogg1* have been linked to an increased cancer risk in humans.
Furthermore, loss of *Ogg1* or *Mutyh* in human neuroblastoma is associated with higher rates of spontaneous C>A mutations.

#### Nonsynonymous mutations may underlie the chromosome 6 mutator phenotype

The p.Thr95Ala variant of the Ogg1 gene is not expected to be harmful according to the SIFT tool, and is found at a nucleotide that is not highly conserved among mammalian species.
The D allele at p.Thr95Ala has a frequency of about 26% in wild-derived Mus musculus domesticus animals, and is fixed in other wild populations of Mus musculus musculus, Mus musculus castaneus, and Mus spretus.
While we would anticipate that Ogg1 deficiency would result in increased 8-oxoG accumulation and higher C>A mutation rates, these findings suggest that p.Thr95Ala may not be very harmful on its own, and may only have a noticeable impact on the BER gene network when Mutyh function is also compromised.
It is also plausible that D alleles at Ogg1 could lead to a subtle increase in C>A mutation rates, but we may not have enough power to detect such a small mutation rate effect in the BXDs.

#### No indication of causal structural variation or mobile element insertions near the chromosome 6 mutator locus

Although we suggest that *Ogg1* is a strong candidate gene for explaining the BXD C>A mutator phenotype, we cannot definitively confirm that the p.Thr95Ala missense mutation is the causal allele.
Initially, we proposed that *Mutyh* missense mutations on *D* haplotypes were responsible for the significant C>A mutator phenotype in the BXDs [@PMID:35545679].
However, further analysis using long-read assemblies of various inbred laboratory mouse strains indicated that this mutator phenotype could be due to a ~5 kbp mobile element insertion (MEI) within the first intron of *Mutyh* [@doi:10.1016/j.xgen.2023.100291].
This MEI was linked to a notable decrease in *Mutyh* expression in embryonic stem cells.
We also examined the new high-quality assemblies to identify any mobile elements or large structural variants (SVs) near the mutator locus on chromosome 6.
However, we did not find convincing evidence suggesting that SVs or MEIs are responsible for the mutator phenotype discussed in this study.

#### Expression quantitative trait loci (eQTLs) might mediate germline mutator phenotypes in the BXDs

We found strong-effect cis-eQTLs for *Ogg1* expression in various tissues in the BXDs (Table @tbl:eqtl-results).
In all these tissue types, *D* genotypes were linked to reduced *Ogg1* expression.
Recent evidence from long-read genome assemblies suggests that an intronic mobile element insertion in *Mutyh* could be responsible for decreased *Mutyh* expression and, consequently, higher C>A mutation rates in BXDs with *D* haplotypes at the chromosome 4 mutator locus [@doi:10.1016/j.xgen.2023.100291].
These findings suggest that the mutator loci on both chromosome 4 and chromosome 6 may increase C>A mutation rates by reducing the expression of DNA repair genes in the same base-excision repair network.


### *Mbd4* may buffer the effects of *Mutyh* mutator alleles by triggering apoptosis

As mentioned in the **Results**, *Mbd4* lies just outside of the 90% bootstrap confidence interval on chromosome 6.
Due to the uncertainties associated with bootstrap confidence intervals in QTL mapping [@PMID:16783000], we have included a discussion of the evidence supporting *Mbd4* as a causal gene below.

In contrast to the *Ogg1* p.Thr95Ala mutation, the p.Asp129Asn variant in *Mbd4* is located within a known protein domain, the *Mbd4* methyl-CpG binding domain.
This variant occurs at a nucleotide and amino acid position that are highly conserved and is anticipated to be harmful according to SIFT (Table @tbl:nonsyn-diffs) [@PMID:12824425].
A similar mutation affecting the corresponding amino acid in humans (p.Asp142Gly in GRCh38/hg38) is found on a single haplotype in the Genome Aggregation Database (gnomAD) and is also forecasted by SIFT and Polyphen to be "deleterious" and "probably_damaging" in human genomes [@PMID:32461654] [@PMID:20354512].

One interesting finding is that mutations in *Mbd4* that result in loss of function do not typically cause an increase in C>A mutations.
Instead, the deficiency of *Mbd4* is usually linked to an increase in C>T mutations at CpG sites.
In our study, we did not observe a higher frequency of C>T mutations in BXDs with *D* alleles at the chromosome 6 mutator locus (see Figure @fig:spectra-comparison-all).
However, research has shown that loss of function mutations in *Mbd4* can worsen the effects of external DNA damage agents.
For instance, studies have found that mouse embryonic fibroblasts with homozygous *Mbd4* knockouts do not undergo apoptosis after exposure to various chemotherapeutic drugs and mutagenic substances.
These external mutagens typically cause DNA damage that is repaired by mismatch repair (MMR) machinery.
Nevertheless, murine intestinal cells with biallelic *Mbd4* LOF mutations also exhibit a reduced apoptotic response to gamma irradiation, which is repaired independently of the MMR gene *Mlh1*.
Furthermore, the complete loss of *Mbd4* function leads to an increased rate of intestinal tumor formation in mice that carry an *Apc* allele associated with intestinal neoplasia.
Additionally, mice with biallelic truncations of the *Mbd4* coding sequence show slightly higher mutation rates in colon cancer cell lines, including increased C>A mutation rates in specific lines.

These findings suggest that the gene *Mbd4* may influence sensitivity to various external factors that cause mutations, possibly by determining whether cells with damaged DNA should die [@PMID:14614141;@PMID:14562041].
It is hypothesized that in mice with a defective repair system for 8-oxoguanine, such as a mutator gene like *Mutyh*, oxidative stress could lead to an increase in DNA damage in the reproductive cells.
If these germ cells have functional *Mbd4* genes, they might be able to induce cell death and partially reduce the impact of a *Mutyh* mutator gene.
However, mice with reduced activity of both *Mbd4* and *Mutyh* may struggle to initiate cell death in response to DNA damage.
Consequently, their germ cells could accumulate higher levels of damage caused by reactive oxygen species, resulting in significantly higher rates of C>A mutations in the germline.

We anticipate that future experimental work will be able to more conclusively establish a mechanistic explanation for the epistatic interaction between mutator loci described in this paper.

### Strengths and limitations of the aggregate mutation spectrum distance approach

Our approach using aggregate mutation spectrum distance (AMSD) successfully identified a mutator allele that was previously overlooked by quantitative trait locus (QTL) mapping.
To compare the effectiveness of AMSD and QTL mapping, we conducted simulations with various parameters.
Our results showed that both methods have similar abilities to detect mutator alleles on haplotypes with numerous new germline mutations.
However, only AMSD was able to locate the mutator locus on chromosome 6 in the BXDs, indicating its superiority over QTL mapping in certain experimental setups.
Simulations also revealed that AMSD is more powerful than QTL mapping when haplotypes have varying numbers of mutations that can aid in mutator mapping.
The BXDs, generated over nearly 40 years in six breeding epochs, have accumulated significantly more mutations in the oldest lines compared to the youngest ones, resulting in noisier mutation spectra.
Unlike QTL mapping, which treats all sample measurements equally, AMSD compares the aggregate mutation spectra of haplotypes at each locus, likely contributing to its enhanced ability to detect mutators in the BXD dataset.

Another advantage of the AMSD approach is that it eliminates the requirement to conduct individual association tests for each potential $k$-mer mutation type, thus avoiding the necessity to modify significance thresholds for multiple tests.
By comparing the entire mutation spectrum between haplotypes containing either allele at a specific site, AMSD is also highly effective in identifying a mutator allele that influences multiple $k$-mer mutation types simultaneously, such as enhancing the frequencies of both C>T and C>A mutations.

<!-- Finally, we find that IHD outperforms QTL mapping when simulated mutators are present at allele frequencies that substantially deviate from 50% ().
This feature of inter-haplotype distance may prove especially useful when mapping mutator alleles in natural, outbreeding populations, in which the effects of negative selection will likely keep mutators at relatively low population frequencies.  -->

However, the AMSD method has some limitations compared to QTL mapping.
Common QTL mapping techniques, such as R/qtl2 (Broman et al., 2019), utilize linear models to assess relationships between genotypes and traits.
This allows for the consideration of various factors, including additive and interactive covariates, as well as kinship matrices, in QTL analyses.
While we have implemented strategies to address relatedness among samples in the AMSD method (see *Materials and Methods*), these approaches are not as versatile as those found in QTL mapping software.
Moreover, the AMSD method assumes that mutator alleles impact a specific subset of mutation types.
If a mutator allele were to uniformly increase mutation rates across all types on the haplotypes it is present in, AMSD would not be able to identify this effect.

### Discovering mutator alleles in other experimental systems

The identification of a second BXD mutator allele highlights the importance of using recombinant inbred lines (RILs) to study the genetic basis of germline mutation rates.
RIL populations are available for many model organisms, and with the decreasing cost of whole-genome sequencing, the AMSD method could be applied to identify mutator alleles in other genetic backgrounds.
However, constructing RIL populations is a time-consuming process that requires significant effort.
If germline mutator alleles can only be identified in these specialized populations, we may only uncover a small portion of the mutator alleles present in natural populations.

Fortunately, the method introduced in this paper can be easily applied to datasets beyond RILs.
Numerous human pedigrees have been sequenced to accurately determine the rate of human *de novo* germline mutation (Campbell et al., 2019; Rahbari et al., 2016; JÃ³nsson et al., 2017).
With the increasing use of family sequencing for diagnosing congenital disorders, these datasets are expanding daily.
Large cohorts of two- or three-generation families provide a scenario where AMSD could be highly effective.
By combining mutation counts from many individuals sharing the same mutator allele, even a subtle mutator signal could potentially be detected amidst the noise of *de novo* germline mutation rate estimates.
It is important to note, however, that modifications will be needed to apply the aggregate mutation spectrum distance approach to outbred, sexually-reproducing cohorts.
AMSD currently assumes individuals have one of two possible genotypes at each marker, without accounting for heterozygous genotypes.
Therefore, the method is currently only suitable for populations like the BXD RILs, where individuals have been inbred long enough for all genotypes to be effectively homozygous.

Selection pressure on mutator alleles in the germline is expected to limit the prevalence of high-impact mutators, although some may still be identifiable through sequencing a significant number of human trios (Smith et al., 2021).
Given that germline mutators typically influence a specific set of mutation types, analyzing mutation spectra may enhance the ability to identify the specific genes responsible for variations in heritable mutation rates, even if each gene has a relatively minor impact on the overall mutation rate per generation (Jones et al., 2019).
