## Materials and Methods

### Identifying *de novo* germline mutations in the BXDs

The BXD resource currently comprises a total of 152 Recombinant Inbred Lines (RILs).
BXDs were derived from either F2 or advanced intercrosses, and subsequently inbred by brother-sister mating for up to 180 generations (Zhu et al., 2021).
BXDs were generated in distinct breeding "epochs," which were each initiated with a distinct cross of C57BL/6J and DBA/2J parents; epochs 1, 2, 4, and 6 were derived from F2 crosses, while epochs 3 and 5 were derived from advanced intercrosses (Zhu et al., 2021).
Previously, we analyzed whole-genome sequencing data from the BXDs and identified candidate *de novo* germline mutations in each line (Zhu et al., 2023). 

$$
\text{A detailed description of the methods used for DNA extraction, sequencing, alignment, and variant processing, as well as the characteristics of the *de novo* mutations, are available in a previous manuscript (Zhu et al., 2023).}
$$

Briefly, we identified private single-nucleotide mutations in each BXD that were absent from all other BXDs, as well as from the C57BL/6J and DBA/2J parents.
Each private variant had to meet the following criteria:

$$
\begin{aligned}
&\text{Genotyped as either homozygous or heterozygous for the alternate allele, with at least 90\% of sequencing reads supporting the alternate allele} \\
&\text{Supported by at least 10 sequencing reads} \\
&\text{Phred-scaled genotype quality of at least 20} \\
&\text{Must not overlap regions of the genome annotated as segmental duplications or simple repeats in GRCm38/mm10} \\
&\text{Must occur on a parental haplotype that was inherited by at least one other BXD at the same locus; these other BXDs must be homozygous for the reference allele at the variant site}
\end{aligned}
$$

This rigorous approach ensured the accuracy and reliability of the identified private mutations.

### A new approach to discover germline mutator alleles

#### Calculating aggregate mutation spectrum distance

We developed a new approach to discover loci that affect the germline *de novo* mutation spectrum in biparental RILs (Figure @fig:distance-method).

We assume that a collection of haplotypes has been genotyped at informative markers, and that *de novo* germline mutations have been identified on each haplotype.

At each informative marker, haplotypes are divided into two groups based on the parental allele they inherited.
A $k$-mer mutation spectrum is then computed using the aggregate mutation counts in each haplotype group.
The $k$-mer mutation spectrum contains the frequency of every possible $k$-mer mutation type in a collection of mutations and can be represented as a vector of size $6 \times 4^{k - 1}$ after collapsing by strand complement.

$$
\text{Important Symbols:}
\begin{align*}
k & : \text{length of the mutation type} \\
\end{align*}
$$

For example, the 1-mer mutation spectrum is a 6-element vector that contains the frequencies of C>T, C>G, C>A, A>G, A>T, and A>C mutations.
Since C>T transitions at CpG nucleotides are often caused by a distinct mechanism (spontaneous deamination of methylated cytosine), the 1-mer mutation spectrum is expanded to include a separate category for CpG>TpG mutations [@PMID:19488047].

At each marker, we computed the cosine distance between the two aggregate spectra.
The cosine distance between two vectors $\mathbf{A}$ and $\mathbf{B}$ is defined as

$$D^C = 1 - \frac{\mathbf{A} \cdot \mathbf{B}}{||\mathbf{A}|| \ ||\mathbf{B}||}$$

where $||\mathbf{A}||$ and $||\mathbf{B}||$ are the $L^2$ (or Euclidean) norms of $\mathbf{A}$ and $\mathbf{B}$, respectively.

The cosine distance metric has several favorable properties for comparing mutation spectra.
It adjusts for the magnitude of the two input vectors, allowing comparison of two spectra with unequal total mutation counts, even if those total counts are relatively small.
Additionally, by calculating the cosine distance between mutation spectra, the need to perform separate comparisons of mutation counts at each individual $k$-mer mutation type is avoided.

Inspired by methods from QTL mapping (Lander and Botstein, 1987; Pritchard et al., 2018), we employed permutation tests to establish genome-wide cosine distance thresholds.
In each of $N$ permutation trials, the per-haplotype mutation data was randomly shuffled so that haplotype labels no longer corresponded to the correct mutation counts.
Using the shuffled mutation data, a genome-wide scan was conducted as described above, and the maximum cosine distance observed at any locus was recorded.
After $N$ permutations (typically 10,000), the $1 - p$ percentile of the distribution of maximum statistics was computed, and that percentile value was used as a genome-wide significance threshold (e.g., at $p = 0.05$).

$$
N: \text{Number of permutation trials} \\
p: \text{Significance level}
$$

#### Estimating confidence intervals around AMSD peaks

If we identified an adjusted cosine distance peak on a particular chromosome, we used a bootstrap resampling approach (Lander and Kruglyak, 1995) to estimate confidence intervals. 

$$
\text{Define symbols:} \\
N = 10,000
$$

In each of $N = 10,000$ trials, we resampled the mutation spectrum data and corresponding marker genotypes (on the chromosome of interest) with replacement.
Using those resampled spectra and genotypes, we performed an aggregate mutation spectrum distance scan on the chromosome of interest and recorded the position of the marker with the largest adjusted cosine distance value. 

$$
\text{Define symbols:} \\
\text{Adjusted cosine distance value} = \text{ACDV}
$$

We then defined a 90% confidence interval by finding two marker locations between which 90% of all $N$ bootstrap samples produced a peak cosine distance value.
In other words, we estimated the bounds of the 90% confidence interval by finding the markers that defined the 5th and 95th percentiles of the distribution of maximum adjusted cosine distance values across $N$ bootstrap trials. 

$$
\text{Define symbols:} \\
\text{90\% confidence interval} = \text{CI}_{90\%}
$$

We note, however, that the bootstrap can exhibit poor performance in QTL mapping studies (Manichaikul et al., 2007); namely, bootstrap confidence intervals tend to be larger than those estimated using either a "LOD drop" method or a Bayes credible interval, and can exhibit poorer-than-expected coverage (a measure of whether the confidence interval contains the true QTL location).

#### Accounting for relatedness between strains 

We expect each BXD to derive approximately 50% of its genome from C57BL/6J and 50% from DBA/2J.
As a result, every pair of BXDs will likely have identical genotypes at a fraction of markers.
Pairs of more genetically similar BXDs may also have more similar mutation spectra, potentially due to shared polygenic effects on the mutation process.
Therefore, at a given marker, if the BXDs that inherited *D* alleles are more genetically dissimilar from those that inherited *B* alleles (considering all loci throughout the genome in our measurement of genetic similarity), we might expect the aggregate mutation spectra in the two groups to also be more dissimilar.

$$
\text{Definition of symbols:} \\
BXD - \text{hybrid mouse strain} \\
C57BL/6J - \text{mouse strain} \\
DBA/2J - \text{mouse strain} \\
D - \text{alleles from DBA/2J} \\
B - \text{alleles from C57BL/6J}
$$

We implemented a simple approach to account for these potential issues of relatedness.
At each marker $g_i$, we divide BXD haplotypes into two groups based on the parental allele they inherited.
As before, we first compute the aggregate mutation spectrum in each group of haplotypes and calculate the cosine distance between the two aggregate spectra ($D^{C}_{i}$). 

$$
D^{C}_{i}
$$

Then, within each group of haplotypes, we calculate the allele frequency of the *D* allele at every marker along the genome to obtain a vector of length $n$, where $n$ is the number of genotyped markers.
To quantify the genetic similarity between the two groups of haplotypes, we calculate the Pearson correlation coefficient $r_i$ between the two vectors of marker-wide *D* allele frequencies.

$$
r_i
$$

Put in another way, at every marker $g_i$ along the genome, we divide BXD haplotypes into two groups and calculate two metrics: $D^{C}_{i}$ (the cosine distance between the two groups' aggregate spectra) and $r_i$ (the correlation between genome-wide *D* allele frequencies in the two groups).
To account for the potential effects of genetic similarity on cosine distances, we perform a regression of $\left(D^C_{1}, D^C_{2}, \ldots D^C_{n} \right)$ on $\left( r_1, r_2, \ldots r_n \right)$ for all $n$ markers using an ordinary least-squares model.
Subsequently, we utilize the residuals from the model as the "adjusted" cosine distance values for each marker.
If genome-wide genetic similarity between haplotypes perfectly predicts cosine distances at each marker, these residuals will all be 0 (or very close to 0).
If genome-wide genetic similarity lacks predictive power, the residuals will simply denote the disparity between the observed cosine distance at a single marker and the marker-wide mean of cosine distances. 

$$
D^{C}_{i} = \frac{\sum_{j=1}^{J} D_{ij}^{A} \cdot D_{ij}^{B}}{\sqrt{\sum_{j=1}^{J} \left(D_{ij}^{A}\right)^2} \cdot \sqrt{\sum_{j=1}^{J} \left(D_{ij}^{B}\right)^2}} {#eq:cosine}
$$

$$
r_i = \frac{\sum_{j=1}^{J} \left(D_{ij}^{A} - \bar{D}_{i}^{A}\right) \cdot \left(D_{ij}^{B} - \bar{D}_{i}^{B}\right)}{\sqrt{\sum_{j=1}^{J} \left(D_{ij}^{A} - \bar{D}_{i}^{A}\right)^2} \cdot \sqrt{\sum_{j=1}^{J} \left(D_{ij}^{B} - \bar{D}_{i}^{B}\right)^2}} {#eq:correlation}
$$

#### Accounting for BXD population structure due to breeding epochs

The current BXD family was generated in six breeding "epochs." Each epoch was initiated with a distinct cross of C57BL/6J and DBA/2J parents.
BXDs in four of the epochs were generated following F2 crosses of C57BL/6J and DBA/2J, while BXDs in the other two were generated following advanced intercrosses.
This breeding approach resulted in differences between the BXD epochs.
For instance, BXDs derived in epochs 3 and 5 (from advanced intercross) harbor larger numbers of fixed recombination breakpoints compared to those from epochs 1, 2, 4, and 6 (Smith et al., 2021).
Although the C57BL/6J and DBA/2J parents used to initialize each epoch were completely inbred, they each possessed a small number of unique *de novo* germline mutations that were subsequently inherited by many of their offspring.
Several of these "epoch-specific" variants have also been associated with phenotypic variation observed between BXDs from different epochs (Smith et al., 2021; Johnson et al., 2019; Brown et al., 2019; White et al., 2020).

To account for potential population structure, as well as epoch-specific effects, we introduced the ability to perform stratified permutation tests in the aggregate mutation spectrum distance approach.
Normally, in each of *N* permutations, we shuffle the per-haplotype mutation spectrum data such that haplotype labels no longer correspond to the correct mutation spectra (i.e., shuffle mutation spectra *across* epochs). 

In the stratified approach, we instead shuffle per-haplotype mutation data *within* epochs, preserving epoch structure while still enabling mutation spectra permutations. 

$$
N = \text{number of permutations}
$$

> We used this epoch-aware approach for all permutation tests presented in this manuscript.


#### Implementation and source code

The aggregate mutation spectrum distance method was implemented in Python, and relies heavily on the following Python libraries: `numpy`, `pandas`, `matplotlib`, `scikit-learn`, `pandera`, `seaborn`, and `numba` [@doi:10.1038/s41586-020-2649-2;@doi:10.5281/zenodo.3509134;@doi:10.1109/MCSE.2007.55;@url:https://jmlr.csail.mit.edu/papers/v12/pedregosa11a.html;@doi:10.25080/Majora-342d178e-010;@doi:10.21105/joss.03021;@doi:10.1145/2833157.2833162].

The code underlying AMSD, as well as documentation of the method, is available on GitHub (https://github.com/quinlan-lab/proj-mutator-mapping).
We have also deposited a reproducible Snakemake [@doi:10.12688/f1000research.29032.1] workflow for running reproducing all analyses and figures presented in the manuscript.

### Simulations to assess the power of the aggregate mutation spectrum distance approach

We performed a series of simple simulations to estimate our power to detect alleles that affect the germline mutation spectrum using the aggregate mutation spectrum distance method.

#### Simulating genotypes 

First, genotypes are simulated on a population of haplotypes at a collection of sites.
A matrix $G$ of size $(s, h)$ is defined, where $s$ represents the number of sites and $h$ represents the number of haplotypes.
It is assumed that each site is biallelic, and the minor allele frequency at each site is 0.5.
For each entry $G_{i,j}$, a single draw is taken from a uniform distribution in the interval $[0.0, 1.0)$.
If the value of the draw is less than 0.5, the value of $G_{i,j}$ is assigned as $1$.
Otherwise, the value of $G_{i,j}$ is assigned as $0$. 

$$
G = \begin{bmatrix}
G_{1,1} & G_{1,2} & \cdots & G_{1,h} \\
G_{2,1} & G_{2,2} & \cdots & G_{2,h} \\
\vdots & \vdots & \ddots & \vdots \\
G_{s,1} & G_{s,2} & \cdots & G_{s,h}
\end{bmatrix}
$$

#### Defining expected mutation type probabilities

Next, we define a vector of 1-mer mutation probabilities:

$$P = \left( 0.29, \ 0.17, \ 0.12, \ 0.075, \ 0.1, \ 0.075, \ 0.17 \right)$$

These probabilities sum to 1 and roughly correspond to the expected frequencies of C>T, CpG>TpG, C>A, C>G, A>T, A>C, and A>G *de novo* germline mutations in mice, respectively [@PMID:31492841].
If we are simulating the 3-mer mutation spectrum, we modify the vector of mutation probabilities $P$ to be length 96, and assign every 3-mer mutation type a value of $\frac{P_c}{16}$, where $P_c$ is the probability of the "central" mutation type associated with the 3-mer mutation type.
In other words, each of the 16 possible N<ins>C</ins>N>N<ins>T</ins>N 3-mer mutation types would be assigned a mutation probability of $\frac{P_c}{16} = \frac{0.46}{16} = 0.02875$.
We then generate a vector of lambda values by scaling the mutation probabilities by the number of mutations we wish to simulate ($m$):

$$
\lambda = Pm
$$

We also create a second vector of lambda values ($\lambda^{\prime}$), in which we multiply the $\lambda$ value of a single mutation type by the mutator effect size $e$.

Rather than simulating the same mean number of mutations ($m$) on every haplotype, a series of simulations were conducted where the mean number of mutations on each haplotype varied.
The BXD RILs were inbred for variable numbers of generations, resulting in each BXD accumulating a variable number of *de novo* germline mutations (Jones et al., 2021).
To better represent the BXD haplotypes, simulations were carried out where the number of mutations ($m$) on each haplotype was randomly selected from a uniform distribution ranging from $m$ to $20m$.
Specifically, a vector of mutation counts $M$ was created, containing $h$ evenly-spaced integers from $m$ to $20m$, where $h$ represents the number of simulated haplotypes.
Therefore, if simulations were performed with a range of 100 to 2,000 mutations on 50 haplotypes, the $i$th entry of $M$ would be $100 + \frac{(2,000 - 100)}{50}i$.
Subsequently, each haplotype's mean number of mutations was determined by referencing the haplotype's index $i$ in $M$. 

$$
m: \text{mean number of mutations on each haplotype}
$$

$$
h: \text{number of simulated haplotypes}
$$


In our simulations, we assume that genotypes at a single site (the "mutator locus") are associated with variation in the mutation spectrum.
That is, at a single site $s_i$, all of the haplotypes with $1$ alleles should have elevated rates of a particular mutation type and draw their mutation counts from $\lambda^{\prime}$, while all of the haplotypes with $0$ alleles should have "wild-type" rates of that mutation type and draw their mutation counts from $\lambda$. 

We therefore pick a random site $s_i$ to be the "mutator locus," and identify the indices of haplotypes in $G$ that were assigned $1$ alleles at $s_i$.
We call these indices $h_{mut}$.

$$
\lambda^{\prime} {#id} - \text{Mutation rate for haplotypes with 1 allele at } s_i
$$

$$
\lambda {#id} - \text{Mutation rate for haplotypes with 0 alleles at } s_i
$$

#### Simulating mutation spectra

To simulate the mutation spectrum on our toy population of haplotypes, we define a matrix $C$ of size $(h, n)$, where $n = 6 \times 4^{k - 1}$ (or if $k = 1$ and we include CpG>TpG mutations, $6 \times 4^{k - 1} + 1$).

Then, we populate the matrix $C$ separately for *mutator* and *wild-type* haplotypes.
For every row $i$ in the matrix (i.e., for every haplotype), we first ask if $i$ is in $h_{mut}$ (that is, if the haplotype at index $i$ was assigned a $1$ allele at the "mutator locus").
If so, we set the values of $C_i$ to be the results of a single Poisson draw from $\lambda^{\prime}$.
If row $i$ is not in $h_{mut}$, we set the values of $C_i$ to be the results of a single Poisson draw from $\lambda$.

$$
C = 
\begin{bmatrix}
C_1 \\
C_2 \\
\vdots \\
C_n
\end{bmatrix}
$$

Here, $C$ is the matrix representing the mutation rates for each haplotype, $h_{mut}$ is the set of haplotypes with the mutator allele, $\lambda$ is the mutation rate for wild-type haplotypes, and $\lambda^{\prime}$ is the

#### Assessing power to detect a simulated mutator allele using AMSD

For each combination of parameters (number of simulated haplotypes, number of simulated markers, mutator effect size, etc.), we conduct 100 independent trials.
Within each trial, we generate the genotype matrix $G$ and the mutation counts $C$.
A "focal" cosine distance is computed as the cosine distance between the aggregate mutation spectra of haplotypes with either genotype at $s_i$ (the site at which an association between genotypes and mutation spectrum variation was artificially induced).
Subsequently, an aggregate mutation spectrum distance scan is carried out using $N = 1,000$ permutations.
If less than 5% of the $N$ permutations result in a cosine distance greater than or equal to the focal distance, we conclude that the method successfully identified the mutator allele in that specific trial.

#### Assessing power to detect a simulated mutator allele using quantitative trait locus (QTL) mapping

Using simulated data, we also evaluated the effectiveness of traditional quantitative trait locus (QTL) mapping in identifying a locus linked to mutation spectrum variability.
As previously mentioned, we generated simulated genotype and mutation spectra for a haplotype population under different conditions, such as the number of mutations per haplotype and the mutator effect size.
With these simulated data, we employed R/qtl2 (Broman et al., 2019) to conduct a genome scan for significant QTL.
In this analysis, we assumed that the simulated genotype markers were evenly distributed along a single chromosome in physical Mbp coordinates.

First, we determined the proportion of each haplotype's *de novo* mutations attributed to each of the $6 \times 4^{k-1}$ potential $k$-mer mutation types.
Subsequently, we converted the simulated genotypes at each marker into genotype probabilities using the `calc_genoprob` function in R/qtl2, utilizing `map_function = "c-f"` and `error_prob = 0`.
For each $k$-mer mutation type, we utilized genotype probabilities and per-haplotype mutation fractions to conduct a QTL scan with the `scan1` function.
To ensure comparability with results from the AMSD method, we refrained from incorporating covariates or kinship matrices in these QTL scans.

Next, we employed the `scan1perm` function to carry out 1,000 permutations of the per-haplotype mutation fractions and compute log-odds (LOD) thresholds for significance.
A QTL scan was deemed "successful" if it yielded a LOD score surpassing the significance threshold (defined using $\alpha = \frac{0.05}{7}$) for the marker where we simulated an association with mutation spectrum variability.

In our simulations, we increase the mutation rate of a single $k$-mer mutation type on haplotypes carrying the simulated mutator allele.
However, in an experimental setting, we would not expect to have prior knowledge of the mutation type affected by the mutator.
Thus, by using an alpha threshold of 0.05 in our simulations, we would likely overestimate the power of QTL mapping for detecting the mutator.
Since we would need to perform 7 separate QTL scans (one for each 1-mer mutation type plus CpG>TpG) in an experimental setting, we calculate QTL LOD thresholds at a Bonferroni-corrected alpha value of $\alpha = \frac{0.05}{7}$. 

$$
\alpha = \frac{0.05}{7} {#eq:alpha}
$$


### Applying the aggregate mutation spectrum distance method to the BXDs

We downloaded previously-generated BXD *de novo* germline mutation data from the GitHub repository associated with our previous manuscript, which was also archived at Zenodo [@url:https://github.com/tomsasani/bxd_mutator_manuscript; @doi:10.5281/zenodo.5941048; @PMID:35545679], and downloaded a CSV file of BXD genotypes at ~7,300 informative markers from GeneNetwork [@url:http://gn1.genenetwork.org/dbdoc/BXDGeno.html; @PMID:27933521].
We also downloaded relevant metadata about each BXD from the manuscript describing the updated BXD resource [@PMID:33472028].
These files are included in the GitHub repository associated with this manuscript.

As in our previous manuscript (Smith et al., 2021), we included mutation data from a subset of the 152 BXDs in our aggregate mutation spectrum distance scans.
Specifically, we removed BXDs that were backcrossed to a C57BL/6J or DBA/2J parent at any point during the inbreeding process (usually, in order to rescue that BXD from inbreeding depression; Jones et al., 2020).
We also excluded BXD68 from our genome-wide scans due to a previously identified hyper-mutator phenotype in that line.
The C>A germline mutation rate in BXD68 is over 5 times the population mean, likely attributed to a private deleterious nonsynonymous mutation in *Mutyh* (Smith et al., 2021).

In our previous manuscript, we eliminated BXDs that had been inbred for fewer than 20 generations.
It takes approximately 20 generations of strict brother-sister mating for a recombinant inbred line (RIL) genome to become >98% homozygous (Johnson et al., 1998).
Therefore, any potential mutator allele would likely be fixed or lost after 20 generations.
If fixed, the allele would remain linked to any excess mutations it causes throughout subsequent inbreeding.
In other words, the *de novo* mutations present in the genome of a "young" BXD (i.e., a BXD inbred for fewer than 20 generations) would not reflect a mutator allele's activity as strongly as mutations in the genome of a much older BXD.
This posed a challenge when using quantitative trait locus mapping to identify mutator alleles in our previous study, as the phenotypes (i.e., C>A mutation rates) of young and old BXDs were given equal weight.
To address this issue, we excluded the younger BXDs from our analysis to prevent the inclusion of their particularly noisy mutation spectra.

Since the Aggregate Mutation Spectrum Distance (AMSD) method calculates an *aggregate* mutation spectrum using all BXDs that inherited a specific allele at a locus, and can mitigate the sparsity and noise of individual mutation spectra, we decided to incorporate these younger BXDs in our genome-wide scans for this study.

In total, we included 117 BXDs in our genome-wide scans.

### Identifying candidate single-nucleotide mutator alleles overlapping the chromosome 6 peak

We investigated the region implicated by our aggregate mutation spectrum distance approach on chromosome 6 by subsetting the joint-genotyped BXD VCF file (European Nucleotide Archive accession PRJEB45429 [1]) using `bcftools` [2].
We defined the candidate interval surrounding the cosine distance peak on chromosome 6 as the 90% bootstrap confidence interval (extending from approximately 95 Mbp to 114 Mbp). 

To predict the functional impacts of both single-nucleotide variants and indels on splicing, protein structure, etc., we annotated variants in the BXD VCF using the following `snpEff` [3] command:

$$
\text{java -jar snpEff.jar -v mm10 BXD.vcf} {#eq:snpEff}
$$

```
 java -Xmx16g -jar /path/to/snpeff/jarfile GRCm38.75 /path/to/bxd/vcf > /path/to/uncompressed/output/vcf
```

and used `cyvcf2` [@PMID:28165109] to iterate over the annotated VCF file in order to identify nonsynonymous fixed differences between the parental C57BL/6J and DBA/2J strains.

### Identifying candidate structural variant alleles overlapping the chromosome 6 peak

We downloaded summary VCFs containing insertion, deletion and inversion structural variants (identified via high-quality, long-read assembly of inbred laboratory mouse strains [@doi:10.1016/j.xgen.2023.100291]) from the Zenodo link associated with the Ferraj et al.
manuscript: https://doi.org/10.5281/zenodo.7644286.

We then downloaded a TSV file containing RefSeq gene predictions in GRCm39/mm39 from the UCSC Table Browser [@PMID:14681465], and used the `bx-python` library [@url:https://github.com/bxlab/bx-python] to intersect the interval spanned by each structural variant with the intervals spanned by the `txStart` and `txEnd` of every RefSeq entry.

We queried all structural variants within the 90% bootstrap confidence interval on chromosome 6.

### Extracting mutation signatures 

We used SigProfilerExtractor (v.1.1.21) [@PMID:30371878] to extract mutation signatures from the BXD mutation data.
After converting the BXD mutation data to the "matrix" input format expected by SigProfilerExtractor, we ran the `sigProfilerExtractor` method as follows:

```python
# install the mm10 mouse reference data
genInstall.install('mm10')

# run mutation signature extraction
sig.sigProfilerExtractor(
    'matrix',
    /path/to/output/directory,
    /path/to/input/mutations,
    maximum_signatures=10,
    nmf_replicates=100,
    opportunity_genome="mm10",
)
```

### Comparing mutation spectra between Mouse Genomes Project strains

We downloaded mutation data from a previously published analysis (Lyon et al., 2019) that identified strain-private mutations in 29 strains that were originally whole-genome sequenced as part of the Sanger Mouse Genomes (MGP) project (Adams et al., 2011).
When comparing counts of each mutation type between MGP strains that harbored either *D* or *B* alleles at the chromosome 4 or chromosome 6 mutator loci, we adjusted mutation counts by the number of callable A, T, C, or G nucleotides in each strain as described previously (Smith et al., 2021).

$$
D: \text{Mutator allele at chromosome 4} \\
B: \text{Mutator allele at chromosome 6}
$$

$$
\text{Equation (@id)}: \text{Mutation count adjustment}
$$

### Querying GeneNetwork for eQTLs at the mutator locus

We used the online GeneNetwork resource (Ghazalpour et al., 2017), which contains array- and RNA-seq-derived expression measurements in a wide variety of tissues, to find *cis*-eQTLs for the DNA repair genes we implicated under the cosine distance peak on chromosome 6.
On the GeneNetwork homepage (genenetwork.org), we selected the "BXD Family" group and used the type dropdown menu to select each of the specific expression datasets described in Table 1.
In the "Get Any" text box, we then entered the listed gene name and clicked "Search".
After selecting the appropriate trait ID on the next page, we used the mapping tools dropdown to run Hayley-Knott regression (Broman and Sen, 2009) with default parameters: 1,000 permutations, interval mapping, no cofactors, and WGS-based genotypes. 

$$
\text{Hayley-Knott regression:}
$$

The Hayley-Knott regression method is a statistical technique used for mapping quantitative trait loci (QTL) in genetic studies.
It is based on a maximum likelihood approach and is commonly used for interval mapping in genome-wide association studies.
The method involves permutation tests to determine the significance of QTL associations.

$$
\text{Equation (1):}
$$

$$
\text{Trait} = \beta_0 + \beta_1 \times \text{Genotype} + \epsilon
$$

If a significant cis-eQTL for the gene of interest was identified (i.e., a locus on chromosome 6 with an LRS greater than or equal to the "significant LRS" genome-wide threshold), a second genome-wide association test for the trait of interest was conducted using GEMMA (Zhou et al., 2014) with the following parameters: WGS-based marker genotypes, a minor allele frequency threshold of 0.05, and leave-one-chromosome-out (LOCO).
By employing both Haley-Knott regression and GEMMA, loci that exceeded a genome-wide LRS threshold could be initially identified, followed by a more precise estimation of the effect of those loci on gene expression (Yao et al., 2021).

$$ LRS = \frac{(LOD)^2}{4 \times \ln(10)} {#eq:LRS} $$

The exact names of the expression datasets we used for each tissue are shown in Table @tbl:eqtl-provenance below:

| Tissue name | Complete name of GeneNetwork expression data | 
| - | - | 
| Kidney | `Mouse kidney M430v2 Sex Balanced (Aug06) RMA` | 
| Gastrointestinal | `UTHSC Mouse BXD Gastrointestinal Affy MoGene 1.0 ST Gene Level (Apr14) RMA` |
| Hematopoetic stem cells | `UMCG Stem Cells ILM6v1.1 (Apr09) transformed` | 
| Spleen | `UTHSC Affy MoGene 1.0 ST Spleen (Dec10) RMA` |  
| Liver | `UTHSC BXD Liver RNA-Seq Avg (Oct19) TPM Log2` | 
| Heart | `NHLBI BXD All Ages Heart RNA-Seq (Nov20) TMP Log2 **` | 
| Hippocampus | `Hippocampus Consortium M430v2 (Jun06) RMA` |

Table: Names of gene expression datasets used for each tissue type on GeneNetwork {#tbl:eqtl-provenance}

### Calculating the frequencies of candidate mutator alleles in wild mice 

To determine the frequencies of the *Ogg1* and *Setmar* nonsynonymous mutations in other populations of mice, a VCF file containing genome-wide variation in 67 wild-derived mice from four species of *Mus* was queried [@PMID:27622383].
The allele frequency of each nonsynonymous mutation in each of the four species or subspecies (*Mus musculus domesticus*, *Mus musculus musculus*, *Mus musculus castaneus*, and *Mus spretus*) was calculated, including genotypes that met the following criteria:

* supported by at least 10 sequencing reads

* Phred-scaled genotype quality of at least 20

### Testing for epistasis between the two mutator loci

To test for statistical epistasis between the mutator loci on chromosome 4 and chromosome 6, we modeled C>A mutation rates in the BXDs as a function of genotypes at either locus. 
Specifically, we tested for statistical interaction between genotypes by fitting a generalized linear model in the R statistical language as follows:

```R
m1 <- glm(Count ~ offset(log(ADJ_AGE)) + Genotype_A * Genotype_B, data = data, family=poisson())
```

In this model, `Count` represents the count of C>A *de novo* mutations observed in each BXD mouse strain.
`ADJ_AGE` is calculated as the product of the number of "callable" cytosine/guanine nucleotides in each BXD (i.e., the total number of cytosines/guanines covered by at least 10 sequencing reads) and the number of generations for which the BXD strain was inbred.
To account for differences in inbreeding duration and genome coverage depth among BXDs, we included the logarithm of `ADJ_AGE` as an "offset" to model the response variable as a mutation rate (expressed per base-pair, per generation) rather than an absolute count.
The genotypes of BXDs at markers `rs27509845` and `rs46276051` are denoted by `Genotype_A` and `Genotype_B`, respectively, which correspond to the markers with peak cosine distances on chromosomes 4 and 6 in the two aggregate mutation spectrum distance scans.
Our analysis focused on the n = 108 BXDs that were homozygous at both sites, allowing us to represent genotypes at each locus as binary variables ("B" or "D").
Through analysis of variance (ANOVA), we compared the model incorporating an interaction effect with a model considering only additive effects:

```R
m2 <- glm(Count ~ offset(log(ADJ_AGE)) + Genotype_A + Genotype_B, data = data, family=poisson())
```

```R
anova(m1, m2, test="Chisq")
```

If model `m1` is a significantly better fit to the data than `m2`, we can reject the null hypothesis that the effect of *D* genotypes at both markers is equal to the sum of the marginal effects of *D* genotypes at either `rs27509845` or `rs46276051`.
In other words, if `m1` is a better fit than `m2`, then the combined effect of *D* genotypes at both markers is non-additive, and indicative of statistical epistasis.

$$
D_{\text{markers}} = D_{\text{rs27509845}} + D_{\text{rs46276051}}
$$

Here, $D_{\text{markers}}$ represents the effect of *D* genotypes at both markers, $D_{\text{rs27509845}}$ represents the effect of *D* genotypes at marker `rs27509845`, and $D_{\text{rs46276051

We tested for epistasis in the Sanger Mouse Genomes Project (MGP) strains using a nearly-identical approach.
In this analysis, we fit two models as follows:

```R
m1 <- glm(Count ~ offset(log(CALLABLE_C)) + Genotype_A * Genotype_B, data = data, family=poisson())

m2 <- glm(Count ~ offset(log(CALLABLE_C)) + Genotype_A + Genotype_B, data = data, family=poisson())
```

where `Count` is the count of strain-private C>A mutations observed in each MGP strain (Yadav et al., 2019).
The `CALLABLE_C` term represents the total number of cytosine and guanine nucleotides that were accessible for mutation calling in each strain, and the `Genotype_A` and `Genotype_B` terms represent MGP genotypes at the chromosome 4 and chromosome 6 mutator loci, respectively.
We compared the two models using ANOVA as described above.

$$ CALLABLE_C = \sum_{i} (C_i + G_i) $$ {#eq:callable_c}

Where:
- $CALLABLE_C$ is the total number of cytosine and guanine nucleotides accessible for mutation calling
- $C_i$ is the number of cytosines at position $i$
- $G_i$ is the number of guanines at position $i$




